source/diffHandler.py

```diff
@@ -168,22 +168,26 @@ def _getText(self, ti: TextInfo) -> str:
 
 def get_dmp_algo():
 	"""
-		This property controls which diff algorithm is used. The default
-		implementation returns either diffHandler.dmp or diffHandler.difflib
-		based on user preference. Subclasses can override this property to
+		This function returns a Diff Match Patch object if allowed by the user.
+		LiveText objects can override the diffAlgo property to
 		choose a diffAlgo object (overriding user preference)
 		if one is incompatible with a particular application.

```

Comment:
This part of the comment still needs work. In particular, the part 'overriding user preference', perhaps you mean "because this approach is new, it can be overridden by user preference, in this case the historical implementation diffLib is used."

If you want to say something specific about LiveText objects, it would be better to move that information there. This file doesn't need to know about LiveText.